arkane/explorer.py

```diff
@@ -193,7 +193,14 @@ def execute(self, outputFile, plot, format='pdf', print_summary=True, speciesLis
             incomplete = False
             for T in Tlist:
                 for P in Plist:
-                    if network.getLeakCoefficient(T=T,P=P) > self.explore_tol:
+                    kchar = 0.0 #compute the characteristic rate coefficient by summing all rate coefficients from the reactant channel
+                    for rxn in self.network.netReactions:#reaction_model.core.reactions+reaction_model.edge.reactions:
+                        if set(rxn.reactants) == set(self.source) and len(rxn.products) == 1 and rxn.products[0].molecule[0].getFormula() == form:
```

Comment:
Can we have `len(rxn.products) == 1` but not `rxn.products[0].molecule[0].getFormula() == form`?

Reply:
Fixed